includes/Common.php

```diff
@@ -0,0 +1,54 @@
+<?php
+namespace WeDevs\WePOS;
+
+if ( ! defined( 'ABSPATH' ) ) {
+    exit;
+}
+
+/**
+* Common Class.
+*/
+class Common {
+
+    /**
+     * Constructor method.
+     */
+    public function __construct() {
+        $this->init_hooks();
+    }
+
+
+    /**
+     * Init hooks method.
+     *
+     * @since WEPOS_LITE_SINCE
+     *
+     * @return void
+     */
+    public function init_hooks() {
+        // Manipulate WooCommerce Order Data.
+        add_action( 'woocommerce_new_order', [ $this, 'set_order_created_via_wepos' ], 10, 2 );
+    }
+
+    /**
+     * Set order created via wepos.
+     *
+     * @since WEPOS_LITE_SINCE
+     *
+     * @param int    $order_id
+     * @param object $order
```

Comment:
it would be more readable if we write like this
```
/**
     * Set order created via wepos.
     *
     * @since WEPOS_LITE_SINCE
     *
     * @param int               $order_id The Order ID.
     * @param \WC_Order $order Order Object.
     *
     * @return void|\WP_Error
     */
```